Nucleation_model.py

```diff
@@ -0,0 +1,89 @@
+#%%
+import datetime as dt
+import shutil
+import math as m
+import numpy as np
+import pandas as pd
+import random as ran
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+from scipy.integrate import solve_ivp
+#%%
+"""=========================== Constants ==========================="""
+
+k_B = 1.380649E-23 #J K-1 // Boltzmann's constant
+ep_0 = 8.854E-12  #F m-1 // electric permittivity of a vacuum
+pi = 3.141529
+N_a = 6.022E23 #mol-1 // Avogadro's number
+R = 8.3145 #J mol-1 K-1 // Ideal constant
+
+"""=========================== User system variables ==========================="""
+#Parameters that can be changed during the trials
+T = 25 + 273.15 #K
+C_LiO2 = 0.15 #mol m-3
+C_Li = 0.15 #mol m-3
+Elyte_v = 0.0005 #m3
+N_0 = 0
+A_0 = pi*0.009**2 #m2
+time = 100000
+
+"""=========================== User thermodyanic imputs ==========================="""
+#parameters that depend on the system being studied
+
```

Comment:
Again, for the section below, give the user detailed info on each input variable, so they know they are giving the correct desired inputs.